packages/compass/src/app/components/home.tsx

```diff
@@ -287,17 +287,10 @@ function Home({
           <CompassInstanceStorePlugin>
             <FieldStorePlugin>
               <ConnectionInfoProvider value={connectionInfo}>
-                {isConnected && connectionInfo && (
-                  <AppRegistryProvider
-                    key={connectionInfo.id}
-                    scopeName="Connected Application"
-                  >
-                    <Workspace
-                      connectionInfo={connectionInfo}
-                      onActiveWorkspaceTabChange={onWorkspaceChange}
-                    />
-                  </AppRegistryProvider>
-                )}
+                <Workspace
+                  connectionInfo={connectionInfo}
```

Comment:
This should be removed right? Connection Info won't be available at the beginning.

Reply:
Yea it won't be but this is to preserve the behaviour of My Queries tab without the feature flag. This will be removed when we move My Queries tab to work without the DS.